goonj/channels/slack_channel.py

```diff
@@ -37,9 +39,10 @@ def send_message(self, sev, message, subject=None, error_id=None,
 
         try:
             headers = {'content-type': 'application/json'}
-            final_text = "Sev: " + sev.value + "\nsubject: " + subject + \
+            final_text = "Sev: " + xstr(sev.value) + "\nsubject: " + xstr(subject) + \
```

Comment:
Use `[''].join()` instead of concatenating using +

http://blog.mclemon.io/python-efficient-string-concatenation-in-python-2016-edition